build/package/.atlascli.goreleaser.yml

```diff
@@ -1,148 +0,0 @@
-# yaml-language-server: $schema=https://goreleaser.com/static/schema-pro.json
```

Comment:
since we are removing this, let's rename `.mongocli.goreleaser.yml` to `.goreleaser.yml`

Reply:
done